frontend/javascripts/viewer/model_initialization.ts

```diff
@@ -695,17 +694,14 @@ function determineDefaultState(
     zoomStep = urlStateZoomStep;
   }
 
-  let rotation = undefined;
-  if (viewMode !== "orthogonal") {
-    rotation = datasetConfiguration.rotation;
-
-    if (someTracing != null) {
-      rotation = Utils.point3ToVector3(someTracing.editRotation);
-    }
+  let rotation = datasetConfiguration.rotation;
+  //TODOM: Now even in ortho mode the default rotation is set. Discuss whether thats what we actually want.
```

Comment:
Jup, thats something we should discuss: Should the default rotation of a dataset configured in the ds settings be applied every time the dataset is viewed / a new annotation is opened (if nothing else is encoded in the URL)

Reply:
yes, I think, this is fine. would be weird, if one wants this but it is not possible.